libs/designsystem/src/lib/components/card/directives/card-is-clickable.directive.ts

```diff
@@ -0,0 +1,16 @@
+import { Directive, OnInit } from '@angular/core';
+
+import { CardComponent } from '../card.component';
+
+@Directive({
+  // tslint:disable-next-line
+  selector: 'kirby-card[click]',
+})
+export class CardIsClickableDirective implements OnInit {
+  constructor(private card: CardComponent) {}
+
+  ngOnInit(): void {
+    this.card.cardIsClickable = true;
+    this.card.flat = false;
```

Comment:
One last thing, to decouple from card, we could make the injected card dependency optional, and make a check here

`if (this.card) {
    this.card.flat = false;
  }`